nightly-benchmark/run-dgx-benchmark.sh

```diff
@@ -22,7 +22,7 @@ which python
 srun -N1 python nightly-run.py > dgx_raw_data.txt
 echo "Copy sitecustomize.py..."
 cp sitecustomize.py ${CONDA_PREFIX}/lib/python3.8/sitecustomize.py
-export DASK_OPTIMIZATION__FUSE__ACTIVE=False
+export DASK_CONFIG="../dask.yaml"
```

Comment:
Guessing we `cd` into this directory. Though happy to change if not

Reply:
Yup!